### DISCUSSION TITLE
OIDC_DISPLAY_NAME gets ignored?

### DISCUSSION THREAD

**Discussion by puyoxyz** [mastodon/mastodon#18554] [Q&A] [answered]
I set up OpenID Connect authentication (with info from [here](https://github.com/mastodon/mastodon/pull/16221)) and it works perfectly, except for one thing: the button says "OPENID_CONNECT"

![image](https://user-images.githubusercontent.com/15311104/170887970-bfb34209-4bb9-44b5-8838-73242905cd56.png)

I've tried `OIDC_DISPLAY_NAME="dummy.cafe account"`, `OIDC_DISPLAY_NAME=dummycafe`, `OIDC_DISPLAY_NAME="dummycafe account"`, `OIDC_DISPLAY_NAME="dummycafe"`, etc etc, but it still says OPENID_CONNECT.

Edit: for anyone who finds this in the future, see https://github.com/mastodon/mastodon/pull/18568

**Comment by ClearlyClaire** [comment, marked as answer]
It does indeed get ignored as far as the button is concerned, and this is an oversight.
